### PR TITLE
Alpine 3.7

### PR DESCRIPTION
**Pull request by cniweb**
This change is [Reviewable](https://reviewable.io/reviews/openhab/openhab-docker/134)

### CODE REVIEW

**Comment by smerschjohann** [comment]
:lgtm:

---

Reviewed 21 of 21 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/openhab/openhab-docker/134#-:-L0ktt-HQpRUBS8-orwV:bnfp4nl)*

File: update.sh [hunk 1]
```diff
@@ -1,341 +1,341 @@
...
-						sed -i "$((line-7)),${line}"d $dstFile
-					fi
-				else
-					cp entrypoint_debian.sh $dstFile
-				fi
-				echo "done"
-		done
-	done
-done
+#!/bin/bash
```
Comment (martinvw) [inline update.sh]: What happened to this file?

**Comment by martinvw** [comment]
The rest looks fine, did you run the / some containers as well?

File: update.sh (hunk 1 above)
Comment (cniweb) [reply update.sh]: wordwrap (Windows / Linux)

**Pull request merged**